changelog_unreleased/html/pr-7556.md

```diff
@@ -0,0 +1,18 @@
+#### Format HTML attribute `style` value ([#7556](https://github.com/prettier/prettier/pull/7556) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div style="   color : red;
+            display    :inline ">
+  </div>
+
+<!-- Prettier stable -->
+<div
+  style="   color : red;
+            display    :inline "
+></div>
+
+<!-- Prettier master -->
+<div style="color: red; display: inline;"></div>
+```
```

Comment:
Also let's put an example with many properties and values, when we don't fix in line

Reply:
done